assets/cubyz/tools/pickaxe.zig.zon

```diff
@@ -38,7 +51,20 @@
 				1.0, 0x0, 0x0, 0.0, 0x0,
 				0x0, 0x0, 0x0, 0x0, 0.0,
 			},
-			.factor = 1.0,
+			.factor = 0.2,
+			.method = .sum,
+		},
+		.{
+			.source = .hardnessDamage,
```

Comment:
I don't see a reason to do this for hardness. The tool doesn't suddently get harder by adding more stuff to it.